docs/book/src/topics/rosa/enabling.md

```diff
@@ -4,5 +4,33 @@ To enable support for ROSA clusters, the ROSA feature flag must be set to true.
 
 ```shell
 export EXP_ROSA="true"
+export EXP_MACHINE_POOL="true"
 clusterctl init --infrastructure aws
+```
+
+## TroubleShoot
+To check the feature-gates for the capi contoller Run the following command.
+```shell
+$ kubectl get deploy capi-controller-manager -n capi-system -o yaml
+```
+check the feature gate container args it should has the MachinePool=true as below.
+
+```yaml
+spec:
+      containers:
+      - args:
+        - --feature-gates=MachinePool=true,ClusterResourceSet=true,ClusterTopology=true,RuntimeSDK=false,MachineSetPreflightChecks=false
+```
+
+To check the feature-gates for the capa contoller Run the following command.
+```shell
+$ kubectl get deploy capa-controller-manager -n capa-system -o yaml
+```
+check the feature gate container args it should has the ROSA=true as below.
+
+```yaml
+spec:
+    containers:
+      - args:
+        - --feature-gates=ROSA=true,...
```

Comment:
change to 2 spacing for better readability?

Reply:
done